From f3f0143ace6866e7370e1cec6ff04f0e6ede11dd Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Wed, 16 Dec 2020 09:28:17 +0000
Subject: [PATCH] overlays: Add missing addresses to ads1015/ads1115

The overlays for the ads1015 and ads1115 I2C ADCs omitted the addresses
in the main device node names. As well as breaking the conventions for
I2C devices, this prevents the firmware from renaming them when the
"reg" property is modified, which in turn stops the overlays from being
instantiated multiple times.

See: https://www.raspberrypi.org/forums/viewtopic.php?f=107&t=294465

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/ads1015-overlay.dts | 2 +-
 arch/arm/boot/dts/overlays/ads1115-overlay.dts | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

--- a/arch/arm/boot/dts/overlays/ads1015-overlay.dts
+++ b/arch/arm/boot/dts/overlays/ads1015-overlay.dts
@@ -13,7 +13,7 @@
             #address-cells = <1>;
             #size-cells = <0>;
             status = "okay";
-            ads1015: ads1015 {
+            ads1015: ads1015@48 {
                 compatible = "ti,ads1015";
                 status = "okay";
                 #address-cells = <1>;
--- a/arch/arm/boot/dts/overlays/ads1115-overlay.dts
+++ b/arch/arm/boot/dts/overlays/ads1115-overlay.dts
@@ -15,7 +15,7 @@
 			#size-cells = <0>;
 			status = "okay";
 
-			ads1115: ads1115 {
+			ads1115: ads1115@48 {
 				compatible = "ti,ads1115";
 				status = "okay";
 				#address-cells = <1>;
